docs/getting-started/quickstart.md

```diff
@@ -79,104 +75,76 @@ def get_contributors(repo_info: dict):
     contributors = response.json()
     return contributors
 
-@flow(name="Repo Info", log_prints=True)  
-def repo_info(
-    repo_owner: str = "PrefectHQ", repo_name: str = "prefect"
-):
-    # call our `get_repo_info` task
+
+@flow(name="Repo Info", log_prints=True)
+def repo_info(repo_owner: str = "PrefectHQ", repo_name: str = "prefect"):
+    """
+    Given a GitHub repository, logs the number of stargazers
+    and contributors for that repo.
+    """
     repo_info = get_repo_info(repo_owner, repo_name)
     print(f"Stars 🌠 : {repo_info['stargazers_count']}")
 
-    # call our `get_contributors` task, 
-    # passing in the upstream result
     contributors = get_contributors(repo_info)
-    print(
-        f"Number of contributors 👷: {len(contributors)}"
-    )
+    print(f"Number of contributors 👷: {len(contributors)}")
 
 
 if __name__ == "__main__":
-    # Call a flow function for a local flow run!
-    repo_info()
+    # create your first deployment
+    repo_info.serve(name="my-first-deployment")
 ```
 
-### Step 4: Run your flow locally
-Call any function that you've decorated with a `@flow` decorator to see a local instance of a flow run.
+Notice that we can write standard Python code within our flow _or_ within our tasks, depending on the level of control and observability we want to give Prefect.
+
+### Step 4: Create a Deployment
+
+When we run this script, Prefect will automatically create a flow deployment that we can interact with via the UI and API; note that this process must stay running in order for your runs to be picked up.
```

Comment:
```suggestion
When we run this script, Prefect will automatically create a flow deployment that you can interact with via the UI and API. The script will say running so that it can listen for scheduled or triggered runs of this flow.
```